### PR TITLE
Disable loading of Custom CSS for atomic sites in `jetpack-mu-wpcom`

### PR DESCRIPTION
**Pull request by monsieur-z**
## Proposed changes:

https://github.com/Automattic/jetpack/pull/37794 caused the wpcomsh plugin to throw the exception: `PHP Fatal error: Cannot redeclare jetpack_register_css_preprocessors()`.

This PR prevents the exception from being thrown by temporarily disabling the loading of the Custom CSS for atomic sites in the `jetpack-mu-wpcom` package (the Custom CSS will still be available in the Jetpack plugin for now). See conversation linked below for more details.
 
### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

p1719591775246119-slack-C05PV073SG3

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

- Spin up a WoA developer site
- Install and activate a classic theme, such as Twenty-Eleven
- From `/wp-admin/admin.php?page=jetpack_modules`, activate the _Custom CSS_ module
- Install the [Jetpack Beta Tester plugin](https://github.com/Automattic/jetpack-beta/releases)
- In WPadmin, go to Jetpack > Beta Tester
- Select this branch for the WordPress.com Features
Screenshot 2024-06-11 at 4 44 34 PM

- In the admin, visit Appearance > Additional CSS
- Make sure you see the options above and that you can write additional CSS

### CODE REVIEW

**Review by anomiex — DISMISSED** [review]
> See conversation linked below for more details.

TL;DR or for any non-Automatticians who wind up here: Jetpack gets updated weekly on WoA sites, while wpcomsh generally gets updated more often. With a version of wpcomsh containing #37794 and a version of Jetpack from earlier this week not containing  #37794, wpcomsh loads first, defines the function, and there isn't yet the guard in Jetpack against redefining it. :boom:

This PR removes the call in wpcomsh that loads all the code added in #37794, so wpcomsh doesn't define the functions. We'll revert this next week after Jetpack gets updated on WoA sites.

**Review by anomiex — APPROVED** [review]
